ENTITY/HasEntityBeenDamagedByEntity.md

```diff
@@ -5,18 +5,15 @@ ns: ENTITY
 
 ```c
 // 0xC86D67D52A707CF8 0x07FC77E0
-BOOL HAS_ENTITY_BEEN_DAMAGED_BY_ENTITY(Entity entity1, Entity entity2, BOOL p2);
+BOOL HAS_ENTITY_BEEN_DAMAGED_BY_ENTITY(Entity entity, Entity damager, BOOL bCheckDamagerVehicle);
 ```
 
-```
-Entity 1 = Victim  
-Entity 2 = Attacker  
-p2 seems to always be 1  
-```
+Checks if the damager entity has damaged the specified entity.
```

Comment:
```suggestion
```

This just duplicates whats in `## Return value`

Reply:
fair point